File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/extraschema/UnusedSchema.java
package io.micronaut.configuration.openapi.docs.extraschema;

// tag::imports[]
import io.micronaut.openapi.annotation.OpenAPIExtraSchema;
// end::imports[]

// tag::clazz[]
@OpenAPIExtraSchema
class UnusedSchema {

    public String field1;
}
// end::clazz[]

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/extraschema/UnusedSchema.kt
package io.micronaut.configuration.openapi.docs.extraschema

// tag::imports[]
import io.micronaut.openapi.annotation.OpenAPIExtraSchema
// end::imports[]

// tag::clazz[]
@OpenAPIExtraSchema
class UnusedSchema {

    var field1: String? = null
}
// end::clazz[]
